### PR TITLE
Add load_shedding example

### PR DESCRIPTION
**Pull request by Marenz**

### CODE REVIEW

File: examples/load_shedding.py [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+                    0,
+                    CONF_STATE.get(self.consumer_name, 0) - self.power_step.as_watts(),
+                )
+                log(
+                    f"Decreased {self.consumer_name} power to {CONF_STATE[self.consumer_name]} W"
+                )
+            else:
+                log("Invalid key. Use 'm' or 'n'.")
+
+    async def _run(self):
```
Comment (matthias-wende-frequenz) [inline examples/load_shedding.py]: Two `_run` methods?

File: examples/load_shedding.py [hunk 2]
```diff
@@ -0,0 +1,243 @@
+import asyncio
```
Comment (matthias-wende-frequenz) [inline examples/load_shedding.py:6]: Header is missing ;)

File: examples/load_shedding.py [hunk 3]
```diff
@@ -0,0 +1,243 @@
...
+
+        for c in consumers:
+            heappush(self.disabled_consumers, c)
+
+    async def _enable_consumer(self, consumer: Consumer):
+        if not consumer.enabled:
+            consumer.enabled = True
+            heappush(self.enabled_consumers, consumer)
+            log(f"+++{consumer.name}, +{consumer.power}")
+            mock_set_consumer(consumer.name, consumer.power.as_watts())
```
Comment (matthias-wende-frequenz) [inline examples/load_shedding.py:134]: I would add a comment that this line is not necessary when using the app in the real word.

File: examples/load_shedding.py [hunk 4]
```diff
@@ -0,0 +1,243 @@
...
+
+    async def _disable_consumer(self, consumer: Consumer):
+        if consumer.enabled:
+            consumer.enabled = False
+            heappush(self.disabled_consumers, consumer)
+            log(f"---{consumer.name}, -{consumer.power}")
+            mock_set_consumer(consumer.name, 0)
```
Comment (matthias-wende-frequenz) [inline examples/load_shedding.py:143]: Same here.

File: examples/load_shedding.py (hunk 1 above)
Comment (Marenz) [reply examples/load_shedding.py]: Wops. how did that happen

File: examples/load_shedding.py [hunk 5]
```diff
@@ -0,0 +1,247 @@
+# License: MIT
+# Copyright © 2023 Frequenz Energy-as-a-Service GmbH
```
Comment (matthias-wende-frequenz) [inline examples/load_shedding.py]: 2024 ;)

**Review by matthias-wende-frequenz — DISMISSED** [review]
A final comment and then lgtm.

**Pull request merged**